manifests/apache.pp

```diff
@@ -107,13 +107,24 @@
       # allow older yum clients to connect, see bz 647828
       custom_fragment            => 'SSLInsecureRenegotiation On',
     }
+
+    # This file is installed by pulp-server but we have everything in the above vhost
+    file {'/etc/httpd/conf.d/pulp.conf':
+      ensure  => file,
+      content => '# This file is managed by the foreman-installer, do not alter.\n',
```

Comment:
Nitpick, but maybe say managed by the puppet-pulp module in case of running outside of foreman-installer